.Rbuildignore

```diff
@@ -9,3 +9,4 @@
 ^\.github$
 ^doc$
 ^data-raw$
+scrape-docs/
```

Comment:
```suggestion
^scrape-docs/
```
Each entry is a regex. Without `^` it will also match `unscrape-docs`.

Reply:
Thanks! Fixed